db/1.9/modules/ROOT/pages/database-connector-query-single.adoc

```diff
@@ -0,0 +1,204 @@
+== Query a Single Record from a Database Example - Mule 4
+
+The *Single Query* operation enables you to execute an SQL query that returns a single record (row) from a database and access the result payload without any additional transformations. This operation does not apply streaming for any of the returned fields of the record. When the query is executed, all the values of the returned row are read into memory and returned in the payload. You must be careful, because if your row is too large you could run out of memory.
+Also, if you are using connection pooling you do not have to worry about your application running out of connections, because this operation returns the connection to the pool immediately after the query is executed.
+
+You can invoke the *Single Query* operation mostly in the same way as you do with the *Select* operation, though you always get a single result regardless of the amount of records returned by the actual SQL query.
+
+The following example shows how to configure Anypoint Connector for Database (Database Connector) *Single Query* operation in a Mule application that retrieves books information from a table in a MySQL database. +
+
+To accomplish this example, you must: +
+
+* Run an SQL statement that creates a table called `BOOK` and inserts 7 records of book information (title, author, bookshopName, isbn, price)
+* Configure the SQL database connection
+* Create the Mule application
+* Run, and test the application with curl commands
+
+The following screenshot shows the Anypoint Studio app flow for this example:
+
+.Query a Single Record from a Database flow
+image::database-query-single-flow.png[Query a Single Record from a Database flow]
+
+== Create the Table in the Database
+
+Before configuring the SQL database connection and creating the Mule application, run the following SQL statement to create a table called `BOOK` and inserts 7 records of books. An incremental `id` is associated to each new book added into the table. For example, the first book added into the table has an `id` equal to `1`, the second book will have an `id` equal to `2`, and so on.
```

Comment:
application > app
and inserts 7 records of book > and insert seven book records